validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -113,7 +114,11 @@ public static ValidatorClientService create(
     final ValidatorRestApiConfig validatorApiConfig = config.getValidatorRestApiConfig();
     Optional<RestApi> validatorRestApi = Optional.empty();
     if (validatorApiConfig.isRestApiEnabled()) {
-      validatorRestApi = Optional.of(ValidatorRestApi.create(validatorApiConfig));
+      validatorRestApi =
+          Optional.of(
+              ValidatorRestApi.create(
+                  validatorApiConfig,
+                  Suppliers.memoize(() -> validatorLoader.getOwnedValidators().getPublicKeys())));
```

Comment:
pretty sure we specifically don't want to use memoize. if we're going this way, GetKeys takes a `Supplier<List<BLSPublicKey>>`
memoize will call that function once, and we're still going to have a single list of keys that can never change as far as I know...